aria/cli/commands.py

```diff
@@ -230,43 +228,48 @@ def _parse(self, uri):
     
     def _get_workflow(self, context, workflow_name):
         if workflow_name in BUILTIN_WORKFLOWS:
-            workflow_fn = import_fullname('aria.orchestrator.workflows.builtin.%s' % workflow_name)
+            workflow_fn = import_fullname('aria.orchestrator.workflows.builtin.{0}'.format(
+                workflow_name))
             inputs = {}
         else:
+            workflow = None
+            for policy in context.modeling.instance.policies:
```

Comment:
if by now there should be a service in the storage, shouldn't this access be to the service's "operations" field instead?